### ISSUE TITLE
Investigate why map svg doesn't work with VoiceOver

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/capstone#498]
hmmmm

**Comment by ChefAndy** [comment]
Is this still the case? It's been changed a few times.
